README.md

```diff
@@ -163,15 +163,18 @@ and energy), Juttle integrates these two layers
 so you don't have to worry about the details
 of wiring Juttle dataflow computation to your browser-based views.
 
-We haven't yet worked out all of the details of this separation, but
-we've put together the components in a prototype test and development utility
-called [outrigger](https://github.com/juttle/outrigger).
-Outrigger pulls in the Juttle core from this repo along with
-the [juttle-viz](https://github.com/juttle/juttle-viz) visualization library and corresponding glue
-to implement Juttle's visualization capabilities.
-Outrigger lets you run Juttle programs from your local file system
+This can be done using [Juttle Engine](https://github.com/juttle/juttle-engine),
```

Comment:
@dmajda Sorry about the rebase... I pushed quickly but not fast enough.

I updated the text to address your comments.